### ISSUE TITLE
When modified external files are detected, 'yes to all' does not refresh the files

### ISSUE THREAD

**Issue by boltex** [leo-editor/leo-editor#3001]
Answering 'yes' works, but 'yes to all' does nothing.

I've tried with the master branch and the devel branch.

**Comment by tbpassin** [comment]
This bug may be related to the following.  In _qt_gui.runAskYesNoDialog_, the "cancel" button entry is commented out but the return values assume that it is present:

```
    return {
        # Buglet: This assumes both yes-all and no-all buttons are active.
        0: 'yes', 1: 'no', 2: 'cancel', 3: 'yes-all', 4: 'no-all',
    }.get(val, 'cancel')
```
So all buttons from 2 on will return the wrong value.

**Comment by tbpassin** [comment]
I corrected the line in question, and after that, "yes-to-all" worked**:
```
    return {
        # Buglet: This assumes both yes-all and no-all buttons are active.
        #  was: 0: 'yes', 1: 'no', 2: 'cancel', 3: 'yes-all', 4: 'no-all',
        0: 'yes', 1: 'no', 2: 'yes-all', 3: 'no-all',
    }.get(val, 'cancel')
```

** Or at least, it saved the one external file I had modified outside Leo.

**Comment by edreamleo** [comment]
@tbpassin Thanks, Thomas, for your investigation. I agree that the assumption about which buttons are active may be the culprit. I'll do my own tests today, keeping in mind #3002.

**Comment by edreamleo** [comment]
Fixed via PR #3013.

**Comment by boltex** [comment]
Thanks !! :smile:
